python/desc/sims/GCRCatSimInterface/HostImages.py

```diff
@@ -0,0 +1,81 @@
+import os
+import numpy as np
+import pandas as pd
+from lsst.sims.utils import angularSeparation
+
+__all__ = ['hostImage']
+
+class hostImage(object):
+
+    def __init__(self, ra_center, dec_center, fov):
+
+        self.ra = ra_center
+        self.dec = dec_center
+        self.radius = fov
+
+
+    def format_catalog(self, df_line, fits_file_name, image_dir):
+
```

Comment:
Please add a docstring to this method explaining what it does and what each of the arguments are.